app/commands/user/become_mentor.rb

```diff
@@ -4,7 +4,7 @@ class User::BecomeMentor
   initialize_with :user, :track_slugs
 
   def call
-    return if user.mentor?
+    return if user.mentor? || user.reputation < 20
```

Comment:
Let's put this 20 into a constant (maybe in `User` called `MIN_REP_TO_MENTOR`) and then update everywhere that's 20 in this PR to `User::MIN_REP_TO_MENTOR`.